Started by user [8mha:////4C8zTO/v0UlG+vQ5Fjm5jTf9kR93yL5yxTMP2X04T9v8AAAAlh+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAyWEgYu/dLi1CL9ovz8EgDKVWzSvwAAAA==[0mroot
Running in Durability level: MAX_SURVIVABILITY
[8mha:////4DOlrZhma2TKYqx8mtRBwGXVE8Xt4z3OJyo/zi+ICDdzAAAAoh+LCAAAAAAAAP9tjTEOwjAQBM8BClpKHuFItIiK1krDC0x8GCfWnbEdkooX8TX+gCESFVvtrLSa5wtWKcKBo5UdUu8otU4GP9jS5Mixv3geZcdn2TIl9igbHBs2eJyx4YwwR1SwULBGaj0nRzbDRnX6rmuvydanHMu2V1A5c4MHCFXMWcf8hSnC9jqYxPTz/BXAFEIGsfuclm8zQVqFvQAAAA==[0m[Pipeline] Start of Pipeline
[8mha:////4B4K7nOhvaaLmbeE3w0NEhE1GAALwLDjakw9plXgkY6XAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycohUghExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jduZBmjwAAAAA==[0m[Pipeline] node
Running on [8mha:////4EheNQ7D4c27QswDGPiDpGtf9rfQF1HauEmyd+/REsZ/AAAAoh+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAyeEgZx/eT83ILSktQiICOvpCg/Jye1SDdLHwCRPP8lzAAAAA==[0mcontroller-j in /root/.jenkins/workspace/spr-test-nicGold
[8mha:////4NUxBMzN6CtFAbQ4pdo7pH8ZuwVHigLf0SdP3FcjQTwyAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gA0xsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfoP95RwAAAAA==[0m[Pipeline] {
[8mha:////4OCB1+Up79PmV3LnMeomKr9r+JFCKkFjAfTpr2UorbLOAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gQkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jc09154wAAAAA==[0m[Pipeline] stage
[8mha:////4N6KQN35K8CT/Pcr9a+R4oOEu4uEh3phvpB1t67zd2hmAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0ggUxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jek7ggRwAAAAA==[0m[Pipeline] { (rm project dir)
[8mha:////4EWjgIQS1R18e39T6LdOu3ZpSMSXMburmKwxJHd8e163AAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gwExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jcChmMxwAAAAA==[0m[Pipeline] script
[8mha:////4NOJ/YVWDlnnBpjDdLKlLZkF5LrR387vblsx7fWhZkPlAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0BAkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfpX/cvwAAAAA==[0m[Pipeline] {
[8mha:////4MAm0moKTc8Eo2PcB0LgkHNHI7Xtf5Poej2Mf/l8xh/lAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbaoyVLaHxBMgiwpL/WWDdrTyRV/MOEjexcpJJ5k3zXm9Y5wQnSpZ7g73DrB2PYbR18YlSfws0cU9XrgkzBcOlmSR15rygpGJgCWtgJWBjUAfKDm2BrfDqodqg0LaXkup3FNC4boAnMFHNRaXyhTnB7j52mfDn+SuAORZg+9pD/AAeoCAqvAAAAA==[0m[Pipeline] sh
+ ls /root/.jenkins/workspace/spr-test-nicGold/npxtester
1.0.0.0001.version
conf
installation
libs
log_analyzer
README.md
testconf
testLibs
Tests
[8mha:////4F3BY6hez6Yn9lSF4dYULOlz4RwK0Fdsp1OYWtj0vxDtAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMKBDglR0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYO0G2F+S7bfcBITL9lL0AAAA=[0m[Pipeline] echo
dir is exists and rm dir
[8mha:////4KL1cTFlC/t8/0LRf71hJwxXIUFLIMviWtcIHXR2W8epAAAAox+LCAAAAAAAAP9tjbEOgjAURS8YB1dHP6KEzcQ4uTYsfkGFWgvNe9g+hMkv8tf8B4kkTt7pnrOc1xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9YJsL8jLsv8AYAPmjb0AAAA=[0m[Pipeline] sh
+ rm -rf /root/.jenkins/workspace/spr-test-nicGold/npxtester
[8mha:////4GzJynz9IX3CLl34e/1JpYOeLz7FDsqMtIihCtFGqNpZAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRVEiIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsTZPsPzjQzob0AAAA=[0m[Pipeline] }
[8mha:////4MvApYhNLg1xEFG2iXyHCog7Z9h4MbdAsNSDoK4wJnDgAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRoqBAVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AO8A7Ay9AAAA[0m[Pipeline] // script
[8mha:////4P1g47DHQeYIMdAQPrqVFJW5XTEYSJkMqP0k+p1upCWxAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIREqJBVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AGnsy8m9AAAA[0m[Pipeline] }
[8mha:////4Lc6WrLYEiiAC7rTWY8q4RQPGu1FD+hrgClvEqpzcg2sAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiKMBZVjZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0g35eCrPwAY5Sibb0AAAA=[0m[Pipeline] // stage
[8mha:////4LPzYIRsJlP9vBl87WR2FbvV5590n9jZkqHCRBijPEs5AAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjJESDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8A6sHc4fBAAAA[0m[Pipeline] stage
[8mha:////4P0yjWfQiEwQxZO7i9obwDXrYwQclNR3i//1CTGbFwBFAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohULDAgpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD1+IXOLCAAAA[0m[Pipeline] { (git clone)
[8mha:////4I3tKNYAz7At397ZkNOs6SFwEn2bzn3ZBhsoegW1dhRwAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKpAQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUG53n9qHNxXfVKy+AAAA[0m[Pipeline] sh
+ git clone -b develop https://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@github.com/intel-innersource/applications.validation.network-platform.npxtester.git npxtester
Cloning into 'npxtester'...
[8mha:////4CRWXV91kUhqh2bTVXI+2n5U32cQ89nJ7yub5blKQ29bAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKoQQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrt/lO7N9Jjga++AAAA[0m[Pipeline] }
[8mha:////4I4ZVrOFMvh2sVfxnfT9s6+paGHoH9OzkxEwdR4h+nRDAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOICFDQoFa3lhheYxBgn1l2wHZyKF/E1/oBFJCq2WO1sM683LIOHmr3BTlNvKTQWBzeavDCx76+OE3Z8wYYpsNModZLc6tOMkqOGOUUJCwErTY3jYMlEWItOPVTlFJnqHH3+jgJK297hCYXI5qh8/MLkYXMb28D08/wVwDQMEcr9Ntfu8AERGGHOvgAAAA==[0m[Pipeline] // stage
[8mha:////4J4BpfnRrLE6odVrNZGgZ11wIRy0HYHBalvcHAcD12j/AAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjEB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INji2eq8EAAAA=[0m[Pipeline] stage
[8mha:////4AgPhGmio4pMNzq5gItmuDxswb3P5dwyIofUIBdMXBTmAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+DOrAycohUdEVMrFEXThCaEJJG/7dJSjpxIq7GHWipxIQHy7ZkvdcbyhjgRMFwp7GzGFvLez+aOfFMobt5ytzRlbeEkbzmjc4NKX1ea0NJw6qCwUbAVmPrKVo0CXbCyYesvERTXVKYt6MAZtUATyjETE4ypG+ZAuzvo4qEP85fAEx9AlYfFquXZ/kBAsTc3cIAAAA=[0m[Pipeline] { (pre-exec-set-args)
[8mha:////4G9dxluK5ut2lxdfo2JVi6OIov1ZH/881VUFzuPzanonAAAAox+LCAAAAAAAAP9tjTESwiAURH/iWNhaeggyE0vHypah8QSYIEKY/wkQSeWJvJp3EM2MlVvs7NvmPV+wjgGOFDSzCgeDsTPMu0mXxTKF4eooM0sX1hFGcooJlQX16rSgoKRgSVXDisNGYecoGtQJttzKu2ycRN2cUyjfgUNt+hEeUPFiTjKkL8wBdrepj4Q/z18BzD5B3baf2vs3w5BRkb4AAAA=[0m[Pipeline] sh
+ python /root/.jenkins/workspace/spr-test-nicGold/npxtester/conf/change_args.py '{"drive_info":{"drive_version":"4.18.0-369.el8.x86_64"},"dpdk_dir":"/opt/APP/utility/DPDK_install/dpdk-22.03","vm_image_conf":{"image1":"rhel86.img","image2":"rhel862.img","image3":"rhel84rt.img","image4":"rhel84rt2.img","image_dir":"/home/image"},"sut_conf":{"host":"10.239.182.80","pf1":"ens43f0","os":"RHEL"},"pkg_generator":{"host":"10.239.182.88","pf1":"ens785f0"}}'
[8mha:////4FhUQONKPXFSeOjFO92QKbtxv93cUzSuhB7hx3xoWX6KAAAApB+LCAAAAAAAAP9tjbEOwiAURV9rHFwd/QiaNG7GyZWw+AXYIkLJexSodPKL/DX/QbSJk3e4uecu5/mCdQxwpKCZVTgYjJ1h3k26LJYpDFdHmVm6sI4wklNMqCyoV6cFBSUFS6oaVhw2CjtH0aBOsOVW3mXjJOrmnEL5Dhxq04/wgIoXc5IhfWEOsLtNfST8ef4KYPY+Qd3uP9W+Ad3NsHm+AAAA[0m[Pipeline] }
[8mha:////4EpM1Tihe33kOTBucrClpdwMCq1FF6wGwTU/AEOZmYaNAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikKgQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrd/lPbN8JP5Su+AAAA[0m[Pipeline] // stage
[8mha:////4JJk4NQMkVX21N+ktknOqKLQw+FD2VU/fvD8dhBPuambAAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjIUSDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8AzYdm7bBAAAA[0m[Pipeline] stage
[8mha:////4GToWcz1lEtttcc1F67+obVKf97ChyGWoNBgdNb8ln6iAAAAqB+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUYoABMXWNsnCC0ISQNvq//KS0EyfiatyBlkpMeLBsS9Z7vaGIDCdiJxqLrcdYe9GF3k1JDMTtNdAgGrqImjBSsELZQZGx1VIVJQuLshxWEtYW60DRo0uwkY1+6DJodOU58bQdJeTe3OEJmZzISXP6lpFhe+tNJPxx/gJg7BLku/1sh/lZfAAyQCqkwgAAAA==[0m[Pipeline] { (run nic case)
[8mha:////4Dqouna/UALz1cds/DYDIib12lRO4R4U9QQgmUx/wyv5AAAAqB+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUYgEJMXWNsnCC0ISQNvq//KS0EyfiatyBlkpMeLBsS9Z7vaGIDCdiJxqLrcdYe9GF3k1JDMTtNdAgGrqImjBSsELZQZGx1VIVJQuLshxWEtYW60DRo0uwkY1+6DJodOU58bQdJeTe3OEJmZzISXP6lpFhe+tNJPxx/gJg7BLku/1sh/lZfAAnLfKUwgAAAA==[0m[Pipeline] script
[8mha:////4JUhH4fIWh5icAEY6zBr3HRHHu0lnMalVyM2fPsu3/3sAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUYgIhpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD0QKVszCAAAA[0m[Pipeline] {
[8mha:////4PJRxj8hD/CuSHgtWDRwAQ67XTdBRcldvWih24U4ysBWAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOKCoEIoFa3lhheYxBgn1l2wHZyKF/E1/oBFJCq2WO1sM683LIOHmr3BTlNvKTQWBzeavDCx76+OE3Z8wYYpsNModZLc6tOMkqOGOUUJCwErTY3jYMlEWItOPVTlFJnqHH3+jgJK297hCYXI5qh8/MLkYXMb28D08/wVwDREKHeHXPvt8AHN/bm5vgAAAA==[0m[Pipeline] sh
+ pytest /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics -s -m nicGold --capture=tee-sys --html=/root/testconfig/testlogs/HtmlReport/nicGold_report.html --self-contained-html
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-6.2.4, py-1.11.0, pluggy-0.13.1
rootdir: /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics, configfile: pytest.ini
plugins: metadata-1.11.0, html-3.1.1
collected 19 items / 15 deselected / 4 selected

npxtester/Tests/testNics/test_mtu.py 2022-04-15 15:47:49 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-15/test_PI_Networking_Jumbo_Packet_CVL_15_47_49/test_PI_Networking_Jumbo_Packet_CVL.log, case_logdir: /root/testconfig/testlogs/2022-04-15/test_PI_Networking_Jumbo_Packet_CVL_15_47_49
2022-04-15 15:47:49 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_Jumbo_Packet_CVL case ============================
2022-04-15 15:47:49 - test_mtu.py - [line:16] - root - INFO - this is Test_mtu setup.
2022-04-15 15:47:49 - test_mtu.py - [line:22] - root - INFO - ssh to the sut1
2022-04-15 15:47:49 - test_mtu.py - [line:24] - root - INFO - ssh to the sut1
2022-04-15 15:47:49 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-15 15:47:49 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-15 15:47:49 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.80
2022-04-15 15:47:49 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-15 15:47:49 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-15 15:47:49 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.88
2022-04-15 15:47:49 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-15 15:47:49 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-15 15:47:52 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-15 15:47:52 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:47:57 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-15 15:47:57 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-15 15:48:00 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-15 15:48:11 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:48:16 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-15 15:48:16 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-15 15:48:19 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-15 15:48:55 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:49:00 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-15 15:49:03 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 up
2022-04-15 15:49:03 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:49:11 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens43f0
2022-04-15 15:49:11 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 4.18.0-369.el8.x86_64
firmware-version: 3.10 0x8000ad67 1.3106.0
expansion-rom-version: 
bus-info: 0000:98:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-15 15:49:19 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 mtu 1500
2022-04-15 15:49:19 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:49:27 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0 mtu 1500
2022-04-15 15:49:27 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:49:35 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0
2022-04-15 15:49:35 - sshInstance.py - [line:54] - root - INFO - ens43f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.10  netmask 255.255.255.255  broadcast 0.0.0.0
        inet6 fe80::6498:6dc9:5596:c914  prefixlen 64  scopeid 0x20<link>
        ether b4:96:91:a0:20:d8  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 24  bytes 3190 (3.1 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-15 15:49:44 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Fri Apr 15 15:49:27 2022 from 10.112.97.60
xhost:  unable to open display ""
[root@CYP10RT ~]# scapy
[39mINFO: Can't import matplotlib. Won't be able to plot.[0m
[39mINFO: Can't import PyX. Won't be able to use psdump() or pdfdump().[0m
[39mINFO: Can't import python-cryptography v1.7+. Disabled WEP decryption/encryption. (Dot11)[0m
[39mINFO: Can't import python-cryptography v1.7+. Disabled IPsec encryption/authentication.[0m
[33m[1mWARNING: IPython not available. Using standard Python shell instead.
AutoCompletion, History are disabled.[0m[0m
[32m[1m                                      [0m[34m[1m[0m
[32m[1m                     aSPY//YASa       [0m[34m[1m[0m
[32m[1m             apyyyyCY//////////YCa    [0m[34m[1m   |[0m
[32m[1m            sY//////YSpcs  scpCY//Pp  [0m[34m[1m   | Welcome to Scapy[0m
[32m[1m ayp ayyyyyyySCP//Pp           syY//C [0m[34m[1m   | Version 2.4.5[0m
[32m[1m AYAsAYYYYYYYY///Ps              cY//S[0m[34m[1m   |[0m
[32m[1m         pCCCCY//p          cSSps y//Y[0m[34m[1m   | https://github.com/secdev/scapy[0m
[32m[1m         SPPPP///a          pP///AC//Y[0m[34m[1m   |[0m
[32m[1m              A//A            cyP////C[0m[34m[1m   | Have fun![0m
[32m[1m              p///Ac            sC///a[0m[34m[1m   |[0m
[32m[1m              P////YCpc           A//A[0m[34m[1m   | Craft me if you can.[0m
[32m[1m       scccccp///pSP///p          p//Y[0m[34m[1m   |                   -- IPv6 layer[0m
[32m[1m      sY/////////y  caa           S//P[0m[34m[1m   |[0m
[32m[1m       cayCyayP//Ya              pY/Ya[0m
[32m[1m        sY/PsY////YCc          aC//Yp [0m
[32m[1m         sc  sccaCY//PCypaapyCP//YSs  [0m
[32m[1m                  spCPY//////YPSps    [0m
[32m[1m                       ccaacs         [0m
[32m[1m                                      [0m
[34m[1m>>> [0m
2022-04-15 15:49:51 - sshInstance.py - [line:75] - root - INFO - sendp([Ether(dst="b4:96:91:a0:20:d8")/IP(len=46)/Raw(load='X'*14480)],iface="ens785f0")
.
Sent 1 packets.
[34m[1m>>> [0m
2022-04-15 15:49:54 - test_mtu.py - [line:92] - root - INFO - success to send package to network
2022-04-15 15:49:58 - sshInstance.py - [line:75] - root - INFO - sendp([Ether(dst="b4:96:91:a0:20:d8")/IP(len=46)/Raw(load='X'*15500)],iface="ens785f0")
Traceback (most recent call last):
  File "/usr/lib64/python3.6/code.py", line 91, in runcode
    exec(code, self.locals)
  File "<console>", line 1, in <module>
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 463, in sendp
    **kargs
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 397, in _send
    realtime=realtime, return_packets=return_packets)
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 360, in __gen_send
    s.send(p)
  File "/usr/local/lib/python3.6/site-packages/scapy/arch/linux.py", line 569, in send
    return SuperSocket.send(self, x)
  File "/usr/local/lib/python3.6/site-packages/scapy/supersocket.py", line 108, in send
    return self.outs.send(sx)
OSError: [Errno 90] Message too long
[34m[1m>>> [0m
2022-04-15 15:50:01 - test_mtu.py - [line:102] - root - INFO - result is right
.2022-04-15 15:50:04 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 mtu 1500
2022-04-15 15:50:04 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:50:12 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0 mtu 1500
2022-04-15 15:50:12 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:50:20 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens43f0/device/sriov_numvfs
2022-04-15 15:50:20 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:50:28 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-15 15:50:28 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:50:33 - test_mtu.py - [line:51] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-15 15:50:33 - test_mtu.py - [line:53] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-15/test_PI_Networking_Jumbo_Packet_CVL_15_47_49
2022-04-15 15:50:36 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-15 15:50:36 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:50:41 - test_mtu.py - [line:55] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-15 15:50:41 - test_mtu.py - [line:56] - root - INFO - this is Test_Nics teardown
2022-04-15 15:50:41 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_Jumbo_Packet_CVL case ============================
2022-04-15 15:50:41 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-15/test_PI_Networking_MTU_set_CVL_15_50_41/test_PI_Networking_MTU_set_CVL.log, case_logdir: /root/testconfig/testlogs/2022-04-15/test_PI_Networking_MTU_set_CVL_15_50_41
2022-04-15 15:50:41 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_MTU_set_CVL case ============================
2022-04-15 15:50:41 - test_mtu.py - [line:16] - root - INFO - this is Test_mtu setup.
2022-04-15 15:50:41 - test_mtu.py - [line:22] - root - INFO - ssh to the sut1
2022-04-15 15:50:41 - test_mtu.py - [line:24] - root - INFO - ssh to the sut1
2022-04-15 15:50:41 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-15 15:50:41 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-15 15:50:41 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.80
2022-04-15 15:50:41 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-15 15:50:41 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-15 15:50:41 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.88
2022-04-15 15:50:41 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-15 15:50:41 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-15 15:50:44 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-15 15:50:45 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:50:50 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-15 15:50:50 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-15 15:50:53 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-15 15:51:03 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:51:08 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-15 15:51:08 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-15 15:51:11 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-15 15:51:47 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:51:52 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-15 15:51:55 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 up
2022-04-15 15:51:55 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:52:03 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens43f0
2022-04-15 15:52:03 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 4.18.0-369.el8.x86_64
firmware-version: 3.10 0x8000ad67 1.3106.0
expansion-rom-version: 
bus-info: 0000:98:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-15 15:52:11 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 mtu 1500
2022-04-15 15:52:11 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:52:19 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0 mtu 2000
2022-04-15 15:52:23 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:52:31 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0
2022-04-15 15:52:31 - sshInstance.py - [line:54] - root - INFO - ens43f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.10  netmask 255.255.255.255  broadcast 0.0.0.0
        inet6 fe80::6498:6dc9:5596:c914  prefixlen 64  scopeid 0x20<link>
        ether b4:96:91:a0:20:d8  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 23  bytes 3094 (3.0 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-15 15:52:40 - sshInstance.py - [line:75] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket

This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register

Last login: Fri Apr 15 16:03:42 2022 from 10.112.97.60
xhost:  unable to open display ""
[root@CYP10RT ~]# scapy
[39mINFO: Can't import matplotlib. Won't be able to plot.[0m
[39mINFO: Can't import PyX. Won't be able to use psdump() or pdfdump().[0m
[39mINFO: Can't import python-cryptography v1.7+. Disabled WEP decryption/encryption. (Dot11)[0m
[39mINFO: Can't import python-cryptography v1.7+. Disabled IPsec encryption/authentication.[0m
[33m[1mWARNING: IPython not available. Using standard Python shell instead.
AutoCompletion, History are disabled.[0m[0m
[32m[1m                                      [0m[34m[1m[0m
[32m[1m                     aSPY//YASa       [0m[34m[1m[0m
[32m[1m             apyyyyCY//////////YCa    [0m[34m[1m   |[0m
[32m[1m            sY//////YSpcs  scpCY//Pp  [0m[34m[1m   | Welcome to Scapy[0m
[32m[1m ayp ayyyyyyySCP//Pp           syY//C [0m[34m[1m   | Version 2.4.5[0m
[32m[1m AYAsAYYYYYYYY///Ps              cY//S[0m[34m[1m   |[0m
[32m[1m         pCCCCY//p          cSSps y//Y[0m[34m[1m   | https://github.com/secdev/scapy[0m
[32m[1m         SPPPP///a          pP///AC//Y[0m[34m[1m   |[0m
[32m[1m              A//A            cyP////C[0m[34m[1m   | Have fun![0m
[32m[1m              p///Ac            sC///a[0m[34m[1m   |[0m
[32m[1m              P////YCpc           A//A[0m[34m[1m   | Craft me if you can.[0m
[32m[1m       scccccp///pSP///p          p//Y[0m[34m[1m   |                   -- IPv6 layer[0m
[32m[1m      sY/////////y  caa           S//P[0m[34m[1m   |[0m
[32m[1m       cayCyayP//Ya              pY/Ya[0m
[32m[1m        sY/PsY////YCc          aC//Yp [0m
[32m[1m         sc  sccaCY//PCypaapyCP//YSs  [0m
[32m[1m                  spCPY//////YPSps    [0m
[32m[1m                       ccaacs         [0m
[32m[1m                                      [0m
[34m[1m>>> [0m
2022-04-15 15:52:47 - sshInstance.py - [line:75] - root - INFO - sendp([Ether(dst="b4:96:91:a0:20:d8")/IP(len=46)/Raw(load='X'*18890)],iface="ens785f0")
.
Sent 1 packets.
[34m[1m>>> [0m
2022-04-15 15:52:50 - test_mtu.py - [line:132] - root - INFO - success to send package to network
2022-04-15 15:52:54 - sshInstance.py - [line:75] - root - INFO - sendp([Ether(dst="b4:96:91:a0:20:d8")/IP(len=46)/Raw(load='X'*20010)],iface="ens785f0")
Traceback (most recent call last):
  File "/usr/lib64/python3.6/code.py", line 91, in runcode
    exec(code, self.locals)
  File "<console>", line 1, in <module>
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 463, in sendp
    **kargs
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 397, in _send
    realtime=realtime, return_packets=return_packets)
  File "/usr/local/lib/python3.6/site-packages/scapy/sendrecv.py", line 360, in __gen_send
    s.send(p)
  File "/usr/local/lib/python3.6/site-packages/scapy/arch/linux.py", line 569, in send
    return SuperSocket.send(self, x)
  File "/usr/local/lib/python3.6/site-packages/scapy/supersocket.py", line 108, in send
    return self.outs.send(sx)
OSError: [Errno 90] Message too long
[34m[1m>>> [0m
2022-04-15 15:52:57 - test_mtu.py - [line:142] - root - INFO - result is right
.2022-04-15 15:53:00 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 mtu 1500
2022-04-15 15:53:00 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:53:08 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0 mtu 1500
2022-04-15 15:53:11 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:53:19 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens43f0/device/sriov_numvfs
2022-04-15 15:53:20 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:53:28 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-15 15:53:28 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:53:33 - test_mtu.py - [line:51] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-15 15:53:33 - test_mtu.py - [line:53] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-15/test_PI_Networking_MTU_set_CVL_15_50_41
2022-04-15 15:53:36 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-15 15:53:36 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:53:41 - test_mtu.py - [line:55] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-15 15:53:41 - test_mtu.py - [line:56] - root - INFO - this is Test_Nics teardown
2022-04-15 15:53:41 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_MTU_set_CVL case ============================
                                  [ 50%]
npxtester/Tests/testNics/test_network_sriov_stress.py 2022-04-15 15:53:41 - NicTestFixture.py - [line:20] - root - INFO - test -------
2022-04-15 15:53:41 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-15/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel86.img-rhel862.img]_15_53_41/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel86.img-rhel862.img].log, case_logdir: /root/testconfig/testlogs/2022-04-15/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel86.img-rhel862.img]_15_53_41
2022-04-15 15:53:41 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel86.img-rhel862.img] case ============================
2022-04-15 15:53:41 - test_network_sriov_stress.py - [line:32] - root - INFO - SSH SUT and setup configurations in the setup function.
2022-04-15 15:53:41 - test_network_sriov_stress.py - [line:50] - root - INFO - ssh to the sut1
2022-04-15 15:53:41 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-15 15:53:41 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-15 15:53:41 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.80
2022-04-15 15:53:41 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-15 15:53:41 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-15 15:53:41 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.88
2022-04-15 15:53:41 - test_network_sriov_stress.py - [line:64] - root - INFO - clean up iperf and taskset process...
2022-04-15 15:53:44 - sshInstance.py - [line:50] - root - INFO - pkill -9 iperf
2022-04-15 15:53:44 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:53:52 - sshInstance.py - [line:50] - root - INFO - pkill -9 taskset
2022-04-15 15:53:52 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:54:00 - sshInstance.py - [line:50] - root - INFO - pkill -9 iperf
2022-04-15 15:54:00 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:54:08 - sshInstance.py - [line:50] - root - INFO - pkill -9 taskset
2022-04-15 15:54:08 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:54:13 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 3] Opened sftp connection (server version 3)
2022-04-15 15:54:13 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-15 15:54:16 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-15 15:54:17 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:54:22 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-15 15:54:22 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-15 15:54:25 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-15 15:54:35 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:54:40 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-15 15:54:40 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-15 15:54:43 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-15 15:55:19 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:55:24 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-15 15:55:27 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 up
2022-04-15 15:55:27 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:55:35 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens43f0
2022-04-15 15:55:35 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 4.18.0-369.el8.x86_64
firmware-version: 3.10 0x8000ad67 1.3106.0
expansion-rom-version: 
bus-info: 0000:98:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-15 15:55:40 - test_network_sriov_stress.py - [line:72] - root - INFO - trying to get ipv6 addr for pf1
2022-04-15 15:55:43 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0
2022-04-15 15:55:43 - sshInstance.py - [line:54] - root - INFO - ens43f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.10  netmask 255.255.255.255  broadcast 0.0.0.0
        inet6 fe80::6498:6dc9:5596:c914  prefixlen 64  scopeid 0x20<link>
        ether b4:96:91:a0:20:d8  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 26  bytes 3351 (3.2 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-15 15:55:51 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0
2022-04-15 15:55:51 - sshInstance.py - [line:54] - root - INFO - ens785f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.11  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::b696:91ff:fea0:2170  prefixlen 64  scopeid 0x20<link>
        ether b4:96:91:a0:21:70  txqueuelen 1000  (Ethernet)
        RX packets 11902767407  bytes 18020190901737 (16.3 TiB)
        RX errors 0  dropped 888  overruns 0  frame 0
        TX packets 484834440  bytes 36210255870 (33.7 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-15 15:55:56 - test_network_sriov_stress.py - [line:569] - root - INFO - 
Stress Disable and Enable for 1 time when VF is Assigned VM guest.

2022-04-15 15:55:59 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-22.03/usertools/dpdk-devbind.py -s | grep ens43f0 | awk '{print $1;}'
2022-04-15 15:56:00 - sshInstance.py - [line:54] - root - INFO - 0000:98:00.0

2022-04-15 15:56:05 - test_network_sriov_stress.py - [line:123] - root - INFO - pf_id was found: 0000:98:00.0

2022-04-15 15:56:05 - test_network_sriov_stress.py - [line:577] - root - INFO - format_pf_id is 0000\:98\:00.0

2022-04-15 15:56:05 - test_network_sriov_stress.py - [line:580] - root - INFO - pf_id_key: 98
2022-04-15 15:56:05 - test_network_sriov_stress.py - [line:584] - root - INFO - start to assign ip for ens43f0 of SUT1: 192.168.10.10
2022-04-15 15:56:11 - sshInstance.py - [line:99] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket
Register this system with Red Hat Insights: insights-client --register
Create an account or view all your systems at https://red.ht/insights-dashboard
Last login: Fri Apr 15 16:07:03 2022 from 10.112.97.60
[root@npx-auto-SPR-quanta ~]# ifconfig ens43f0 192.168.10.10
[root@npx-auto-SPR-quanta ~]# 
2022-04-15 15:56:14 - test_network_sriov_stress.py - [line:326] - root - INFO - Successfully assigned ip to ethernet: ens43f0
2022-04-15 15:56:14 - test_network_sriov_stress.py - [line:586] - root - INFO - start to assign ip for ens785f0 of SUT2: 192.168.10.11
2022-04-15 15:56:20 - sshInstance.py - [line:99] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Fri Apr 15 16:06:38 2022 from 10.112.97.60
xhost:  unable to open display ""
[root@CYP10RT ~]# ifconfig ens785f0 192.168.10.11
[root@CYP10RT ~]# 
2022-04-15 15:56:23 - test_network_sriov_stress.py - [line:326] - root - INFO - Successfully assigned ip to ethernet: ens785f0
2022-04-15 15:56:23 - test_network_sriov_stress.py - [line:129] - root - INFO - virtulize sut1 pf to 4 vfs
2022-04-15 15:56:26 - sshInstance.py - [line:50] - root - INFO - echo 4 > /sys/class/net/ens43f0/device/sriov_numvfs
2022-04-15 15:56:30 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:56:35 - test_network_sriov_stress.py - [line:138] - root - INFO - Successfully Virtualized PF to VFs.
2022-04-15 15:56:38 - sshInstance.py - [line:50] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-04-15 15:56:38 - sshInstance.py - [line:54] - root - INFO - 98:01.0
98:01.1
98:01.2
98:01.3

2022-04-15 15:56:43 - test_network_sriov_stress.py - [line:152] - root - INFO - Succssfully Gotten vf_id list:

2022-04-15 15:56:46 - sshInstance.py - [line:50] - root - INFO - modprobe vfio-pci

2022-04-15 15:56:46 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:56:51 - test_network_sriov_stress.py - [line:172] - root - INFO - start to bind drive in sut1
2022-04-15 15:56:54 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-22.03/usertools/dpdk-devbind.py -b vfio-pci 98:01.0
2022-04-15 15:56:54 - sshInstance.py - [line:54] - root - INFO - Warning: routing table indicates that interface 0000:98:01.0 is active. Not modifying

2022-04-15 15:56:59 - test_network_sriov_stress.py - [line:186] - root - INFO - Successfuly bound VF to vfio-pci
2022-04-15 15:57:02 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-22.03/usertools/dpdk-devbind.py -b vfio-pci 98:01.1
2022-04-15 15:57:03 - sshInstance.py - [line:54] - root - INFO - Warning: routing table indicates that interface 0000:98:01.1 is active. Not modifying

2022-04-15 15:57:08 - test_network_sriov_stress.py - [line:186] - root - INFO - Successfuly bound VF to vfio-pci
2022-04-15 15:57:08 - test_network_sriov_stress.py - [line:604] - root - INFO - lspci -Dnn | grep 98:01.1
2022-04-15 15:57:11 - sshInstance.py - [line:50] - root - INFO - lspci -Dnn | grep 98:01.1
2022-04-15 15:57:11 - sshInstance.py - [line:54] - root - INFO - 0000:98:01.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

2022-04-15 15:57:16 - test_network_sriov_stress.py - [line:616] - root - INFO - Successfully get vf device id: 8086:1889
2022-04-15 15:57:16 - test_network_sriov_stress.py - [line:209] - root - INFO - get default network info: virsh net-info default | grep "Active" | awk '{print $2;}'
2022-04-15 15:57:19 - sshInstance.py - [line:50] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-04-15 15:57:19 - sshInstance.py - [line:54] - root - INFO - yes

2022-04-15 15:57:24 - test_network_sriov_stress.py - [line:212] - root - INFO - Default network is active? yes

2022-04-15 15:57:24 - test_network_sriov_stress.py - [line:629] - root - INFO - start to create VMs....
2022-04-15 15:57:24 - test_network_sriov_stress.py - [line:242] - root - INFO - try to create VM with the cmd1: virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_98_01_0 --boot hd --disk path=/home/image/rhel86.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.0.0,port=5900 --noautoconsole

2022-04-15 15:57:28 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host ---vcpus=5 --network network=default --hostdev pci_0000_98_01_0 --boot hd --disk ppath=/home/image/rhel86.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,,listen=0.0.0.0,port=5900 --noautoconsole
2022-04-15 15:57:30 - sshInstance.py - [line:75] - root - INFO - 
Starting install...
2022-04-15 15:57:34 - sshInstance.py - [line:75] - root - INFO - Domain creation completed.
[root@npx-auto-SPR-quanta ~]# 
2022-04-15 15:57:37 - test_network_sriov_stress.py - [line:253] - root - INFO - Successfully created a VM: VMtest1
2022-04-15 15:57:37 - test_network_sriov_stress.py - [line:242] - root - INFO - try to create VM with the cmd1: virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_98_01_1 --boot hd --disk path=/home/image/rhel862.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.0.0,port=5901 --noautoconsole

2022-04-15 15:57:41 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host ---vcpus=5 --network network=default --hostdev pci_0000_98_01_1 --boot hd --disk ppath=/home/image/rhel862.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vncc,listen=0.0.0.0,port=5901 --noautoconsole
2022-04-15 15:57:43 - sshInstance.py - [line:75] - root - INFO - 
Starting install...
2022-04-15 15:57:48 - sshInstance.py - [line:75] - root - INFO - Domain creation completed.
[root@npx-auto-SPR-quanta ~]# 
2022-04-15 15:57:51 - test_network_sriov_stress.py - [line:253] - root - INFO - Successfully created a VM: VMtest2
2022-04-15 15:57:54 - sshInstance.py - [line:50] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-04-15 15:57:55 - sshInstance.py - [line:54] - root - INFO - 192.168.122.138/24

2022-04-15 15:58:00 - test_network_sriov_stress.py - [line:269] - root - INFO - Successfully got VMtest1 IP: 192.168.122.138/24

2022-04-15 15:58:03 - sshInstance.py - [line:50] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-04-15 15:58:03 - sshInstance.py - [line:54] - root - INFO - 192.168.122.76/24

2022-04-15 15:58:08 - test_network_sriov_stress.py - [line:269] - root - INFO - Successfully got VMtest2 IP: 192.168.122.76/24

2022-04-15 15:58:08 - test_network_sriov_stress.py - [line:639] - root - INFO - ip of VM1: 192.168.122.138
2022-04-15 15:58:08 - test_network_sriov_stress.py - [line:640] - root - INFO - ip of VM1: 192.168.122.76
2022-04-15 15:58:08 - test_network_sriov_stress.py - [line:645] - root - INFO - Start ssh VM1 and conf VM1
2022-04-15 15:58:12 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.122.138
PING 192.168.122.138 (192.168.122.138) 56(84) bytes of data.
64 bytes from 192.168.122.138: icmp_seq=1 ttl=64 time=0.135 ms
2022-04-15 15:58:13 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.138: icmp_seq=2 ttl=64 time=0.069 ms
2022-04-15 15:58:14 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.138: icmp_seq=3 ttl=64 time=0.056 ms
2022-04-15 15:58:15 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.138: icmp_seq=4 ttl=64 time=0.065 ms
2022-04-15 15:58:16 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.138: icmp_seq=5 ttl=64 time=0.079 ms

--- 192.168.122.138 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4118ms
rtt min/avg/max/mdev = 0.056/0.080/0.135/0.030 ms
[root@npx-auto-SPR-quanta ~]# 
2022-04-15 15:58:25 - sshInstance.py - [line:99] - root - INFO - ssh root@192.168.122.138
The authenticity of host '192.168.122.138 (192.168.122.138)' can't be established.
ECDSA key fingerprint is SHA256:J52wv4LPVprqJsqIIJZZQXHlsApVTL3W7ooy490//CA.
Are you sure you want to continue connecting (yes/no/[fingerprint])? 
2022-04-15 15:58:34 - sshInstance.py - [line:99] - root - INFO - yes
Warning: Permanently added '192.168.122.138' (ECDSA) to the list of known hosts.
root@192.168.122.138's password: 
2022-04-15 15:58:43 - sshInstance.py - [line:99] - root - INFO - 
Activate the web console with: systemctl enable --now cockpit.socket
Register this system with Red Hat Insights: insights-client --register
Create an account or view all your systems at https://red.ht/insights-dashboard
Last login: Thu Apr 14 22:19:50 2022 from 192.168.122.1
[root@philliph-IVM ~]# 
2022-04-15 15:58:46 - test_network_sriov_stress.py - [line:352] - root - INFO - Successfully ssh VM.
2022-04-15 15:58:52 - sshInstance.py - [line:99] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $$1;}'
0000:05:00.0
[root@philliph-IVM ~]# 
2022-04-15 15:58:58 - test_network_sriov_stress.py - [line:369] - root - INFO - successfully got vf pci id of vm1: 0000:05:00.0
2022-04-15 15:59:04 - sshInstance.py - [line:99] - root - INFO - ls /sys/bus/pci/devices/0000\:05\:00.0/net/ --color=never [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            r
enp5s0
[root@philliph-IVM ~]# 
2022-04-15 15:59:09 - test_network_sriov_stress.py - [line:383] - root - INFO - Successfully got vf device name of vm1: enp5s0
2022-04-15 15:59:15 - sshInstance.py - [line:99] - root - INFO - ifconfig enp5s0 192.168.10.12
[root@philliph-IVM ~]# 
2022-04-15 15:59:18 - test_network_sriov_stress.py - [line:326] - root - INFO - Successfully assigned ip to ethernet: enp5s0
2022-04-15 15:59:24 - sshInstance.py - [line:99] - root - INFO - ifconfig enp5s0 down
[root@philliph-IVM ~]# 
2022-04-15 15:59:33 - sshInstance.py - [line:99] - root - INFO - ifconfig enp5s0 hw ether 1234567890ac
[root@philliph-IVM ~]# 
2022-04-15 15:59:42 - sshInstance.py - [line:99] - root - INFO - ifconfig enp5s0 up
[root@philliph-IVM ~]# 
2022-04-15 15:59:51 - sshInstance.py - [line:99] - root - INFO - nmcli device connect enp5s0
[2KDevice 'enp5s0' successfully activated with '1f069b77-7a05-45a6-bec1-3e45ca49d66e'.
[root@philliph-IVM ~]# 
2022-04-15 16:00:00 - sshInstance.py - [line:99] - root - INFO - ifconfig enp5s0
enp5s0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 27  bytes 2826 (2.7 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
[root@philliph-IVM ~]# 
2022-04-15 16:00:09 - sshInstance.py - [line:99] - root - INFO - exit
logout
Connection to 192.168.122.138 closed.
[root@npx-auto-SPR-quanta ~]# 
F2022-04-15 16:00:12 - test_network_sriov_stress.py - [line:84] - root - INFO - removing VMs: VMtest1, VMtest2
2022-04-15 16:00:15 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest2
2022-04-15 16:00:15 - sshInstance.py - [line:54] - root - INFO - Domain 'VMtest2' destroyed


2022-04-15 16:00:20 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully shut down the VM: VMtest2
2022-04-15 16:00:23 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest2
2022-04-15 16:00:23 - sshInstance.py - [line:54] - root - INFO - Domain 'VMtest2' has been undefined


2022-04-15 16:00:28 - test_network_sriov_stress.py - [line:311] - root - INFO - Successfully removed the VM: VMtest2
2022-04-15 16:00:31 - sshInstance.py - [line:50] - root - INFO - virsh destroy VMtest1
2022-04-15 16:00:32 - sshInstance.py - [line:54] - root - INFO - Domain 'VMtest1' destroyed


2022-04-15 16:00:37 - test_network_sriov_stress.py - [line:301] - root - INFO - Successfully shut down the VM: VMtest1
2022-04-15 16:00:40 - sshInstance.py - [line:50] - root - INFO - virsh undefine VMtest1
2022-04-15 16:00:40 - sshInstance.py - [line:54] - root - INFO - Domain 'VMtest1' has been undefined


2022-04-15 16:00:45 - test_network_sriov_stress.py - [line:311] - root - INFO - Successfully removed the VM: VMtest1
2022-04-15 16:00:48 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens43f0/device/sriov_numvfs
2022-04-15 16:00:56 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:01:04 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 0
2022-04-15 16:01:04 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:01:12 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0 0
2022-04-15 16:01:12 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:01:20 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-15 16:01:20 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:01:25 - test_network_sriov_stress.py - [line:93] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-15 16:01:25 - test_network_sriov_stress.py - [line:95] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-15/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel86.img-rhel862.img]_15_53_41
2022-04-15 16:01:28 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-15 16:01:28 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:01:33 - test_network_sriov_stress.py - [line:97] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-15 16:01:36 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/*.txt
2022-04-15 16:01:36 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:01:41 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-15 16:01:41 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel86.img-rhel862.img] case ============================
                  [ 75%]
npxtester/Tests/testNics/test_nics.py 2022-04-15 16:01:41 - NicTestFixture.py - [line:20] - root - INFO - test -------
2022-04-15 16:01:41 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-15/test_gold_PI_Networking_SRIOV_CheckDevice_16_01_41/test_gold_PI_Networking_SRIOV_CheckDevice.log, case_logdir: /root/testconfig/testlogs/2022-04-15/test_gold_PI_Networking_SRIOV_CheckDevice_16_01_41
2022-04-15 16:01:41 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_gold_PI_Networking_SRIOV_CheckDevice case ============================
2022-04-15 16:01:41 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-15 16:01:41 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-15 16:01:41 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-15 16:01:41 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-15 16:01:41 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.80
2022-04-15 16:01:41 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-15 16:01:41 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-15 16:01:41 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.88
2022-04-15 16:01:41 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-15 16:01:44 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 up
2022-04-15 16:01:44 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:01:52 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens43f0 | grep ^version:
2022-04-15 16:01:52 - sshInstance.py - [line:54] - root - INFO - version: 4.18.0-369.el8.x86_64

2022-04-15 16:01:57 - test_nics.py - [line:58] - root - INFO - Passed: driver version is what expected.
2022-04-15 16:01:57 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-15 16:02:00 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-15 16:02:00 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:02:05 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-15 16:02:05 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-15 16:02:08 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-15 16:02:19 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:02:24 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-15 16:02:24 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-15 16:02:27 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-15 16:03:02 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:03:07 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-15 16:03:10 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 up
2022-04-15 16:03:10 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:03:18 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens43f0
2022-04-15 16:03:18 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 4.18.0-369.el8.x86_64
firmware-version: 3.10 0x8000ad67 1.3106.0
expansion-rom-version: 
bus-info: 0000:98:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-15 16:03:26 - sshInstance.py - [line:50] - root - INFO - modprobe iavf
2022-04-15 16:03:27 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:03:32 - test_nics.py - [line:65] - root - INFO - --Finish modprobe iavf --
2022-04-15 16:03:35 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0
2022-04-15 16:03:35 - sshInstance.py - [line:54] - root - INFO - ens43f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.10  netmask 255.255.255.255  broadcast 0.0.0.0
        inet6 fe80::6498:6dc9:5596:c914  prefixlen 64  scopeid 0x20<link>
        ether b4:96:91:a0:20:d8  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 22  bytes 3024 (2.9 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-15 16:03:43 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0
2022-04-15 16:03:43 - sshInstance.py - [line:54] - root - INFO - ens785f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::b696:91ff:fea0:2170  prefixlen 64  scopeid 0x20<link>
        ether b4:96:91:a0:21:70  txqueuelen 1000  (Ethernet)
        RX packets 11902767571  bytes 18020190921635 (16.3 TiB)
        RX errors 0  dropped 908  overruns 0  frame 0
        TX packets 484834442  bytes 36210255990 (33.7 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-15 16:03:51 - sshInstance.py - [line:50] - root - INFO - echo 4 > /sys/class/net/ens43f0/device/sriov_numvfs
2022-04-15 16:03:55 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:04:08 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0 192.168.10.11

2022-04-15 16:04:08 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:04:21 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0v0 down
2022-04-15 16:04:21 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:04:29 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0v0 up
2022-04-15 16:04:29 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:04:37 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0v0 192.168.10.12
2022-04-15 16:04:37 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:04:48 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.12
2022-04-15 16:05:03 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 0 received, 100% packet loss, time 4129ms


F2022-04-15 16:05:14 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens43f0/device/sriov_numvfs
2022-04-15 16:05:21 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:05:29 - sshInstance.py - [line:50] - root - INFO - echo 0 > /sys/class/net/ens43f0/device/sriov_numvfs
2022-04-15 16:05:29 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:05:37 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 0
2022-04-15 16:05:37 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:05:45 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0 0
2022-04-15 16:05:45 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:05:53 - sshInstance.py - [line:50] - root - INFO - dmesg > /tmp/dmesg.log
2022-04-15 16:05:53 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:05:58 - test_nics.py - [line:89] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-04-15 16:05:58 - test_nics.py - [line:91] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-04-15/test_gold_PI_Networking_SRIOV_CheckDevice_16_01_41
2022-04-15 16:06:01 - sshInstance.py - [line:50] - root - INFO - rm -f /tmp/dmesg.log
2022-04-15 16:06:01 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:06:06 - test_nics.py - [line:93] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-04-15 16:06:06 - sshInstance.py - [line:43] - root - INFO - closing ssh client on SUT
2022-04-15 16:06:06 - test_nics.py - [line:95] - root - INFO - this is Test_Nics teardown
2022-04-15 16:06:06 - NicTestFixture.py - [line:35] - root - INFO - ============================ end of run test_gold_PI_Networking_SRIOV_CheckDevice case ============================
                                  [100%]

=================================== FAILURES ===================================
_ Test_Network_SRIOV_Stress.test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel86.img-rhel862.img] _

self = <Tests.testNics.test_network_sriov_stress.Test_Network_SRIOV_Stress object at 0x7f9a8e504ef0>
vm1_image = 'rhel86.img', vm2_image = 'rhel862.img'

    @pytest.mark.usefixtures("fbefore")
    @pytest.mark.nicAll
    @pytest.mark.nicGold
    @pytest.mark.nicSriov5times
    @pytest.mark.parametrize('vm1_image, vm2_image', vm_image)
    def test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest(self, vm1_image, vm2_image):
        for j in range(1, 2):
            logging.info("\nStress Disable and Enable for {} time when VF is Assigned VM guest.\n".format(j))
            # get pf id: 0000:xx:xx.x
            pf_id, status = self.get_pf_id(self.client, self.pf_name)
            # transfer pf_id to 0000\:00\:00.0 format
            if not pf_id:
                assert 200 == status
            else:
                pf_id_format = pf_id.replace(":", "\\:", 2)
                logging.info("format_pf_id is {}".format(pf_id_format))
                # get the number key of pf_id
                pf_id_key = pf_id.split(":")[1]
                logging.info("pf_id_key: {}".format(pf_id_key))
                # assign IP to PF in SUT and sut2
                pf_ip = "192.168.10.10"
                sut2_ip = "192.168.10.11"
                logging.info("start to assign ip for {} of SUT1: {}".format(self.pf_name, pf_ip))
                assert 200 == self.assign_ip_ehternet(self.session, pf_ip, self.pf_name)
                logging.info("start to assign ip for {} of SUT2: {}".format(self.pf2_name, sut2_ip))
                assert 200 == self.assign_ip_ehternet(self.session2, sut2_ip, self.pf2_name)
                # virtulize the pf to vfs
                assert 200 == self.virtualize_pf_to_vfs(self.pf_name)
                # get vf ids
                vf_id_list, status = self.get_vf_ids(self.client, pf_id_key)
                vf_id_list = [i.strip() for i in vf_id_list]
                if status == 200:
                    # modprobe vfio-pci
                    assert 200 == self.modprobe("vfio-pci")
                    # bind vf with vfio-pci
                    host_dev_id_list, status = self.bind_vf_to_vfio_pci(self.client, vf_id_list)
                    assert 200 == status
                else:
                    assert 200 == status
                # get vF device ida
                # tmp code
                cmd = "lspci -Dnn | grep %s" % (vf_id_list[1])
                logging.info(cmd)
                vf_device_id = ""
                vf_device_id_rs, error = self.ssh_sut.ssh_exec_cmd(self.client, cmd)
                vf_device_id_rs = vf_device_id_rs.strip()
                if vf_device_id_rs:
                    st = False
                    vf_device_id = None
                    for one in vf_device_id_rs.split(" "):
                        one = " " + one + " "
                        if one.find(" [") > -1 and one.find("] ") > -1 and one.find(":") > -1:
                            vf_device_id = one.replace("[", "").replace("]", "").strip()
                    if not vf_device_id: assert 200 == 400, "vf_device_id is not found"
                    logging.info("Successfully get vf device id: {}".format(vf_device_id))
                    assert 200 == 200
                else:
                    logging.error("Fail to get vf device id.")
                    assert 200 == 400
    
                # set network
                status = self.default_network_isActive(self.client, self.defult_network)
                if status == "no":
                    logging.info("start to check default network status...")
                    assert 200 == self.start_default_network(self.client, self.defult_network)
    
                # start VMs
                logging.info("start to create VMs....")
                if self.sut_info["os"] == "UBUNTU":
                    assert 200 == self.create_vm(vm1_image, host_dev_id_list[0],  self.vm1_name, "ubuntu20.04", "5900")
                    assert 200 == self.create_vm(vm2_image, host_dev_id_list[1], self.vm2_name, "ubuntu20.04", "5901")
                else:
                    assert 200 == self.create_vm(vm1_image, host_dev_id_list[0], self.vm1_name, "rhel8.2", "5900")
                    assert 200 == self.create_vm(vm2_image, host_dev_id_list[1], self.vm2_name, "rhel8.2", "5901")
                # get VM ssh IP
                vm1_ip, status1 = self.get_vm_ip(self.vm1_name)
                vm2_ip, status2 = self.get_vm_ip(self.vm2_name)
                logging.info("ip of VM1: {}".format(vm1_ip))
                logging.info("ip of VM1: {}".format(vm2_ip))
    
                # ssh VM1 to set VF
                vm1_vf_ip = "192.168.10.12"
                vm2_vf_ip = "192.168.10.13"
                logging.info("Start ssh VM1 and conf VM1")
                assert 200 == self.on_session_ssh(self.ssh_sut, self.session,
                                                  vm1_ip), "ssh vm failed,the reason may be is password is error,vm ip isimpassability"
                vm1_vf_name, vm1_ipv6, status = self.ssh_vm_set_VF(vm1_ip, vf_device_id, vm1_vf_ip, "1234567890ac")
                self.ssh_sut.run_ssh_cmd(self.session, "exit\n")
>               assert 200 == status, "vm1 vf related settings failed"
E               AssertionError: vm1 vf related settings failed
E               assert 200 == 400

npxtester/Tests/testNics/test_network_sriov_stress.py:650: AssertionError
---------------------------- Captured stdout setup -----------------------------
2022-04-15 15:53:41 - NicTestFixture.py - [line:20] - root - INFO - test -------
2022-04-15 15:53:41 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-15/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel86.img-rhel862.img]_15_53_41/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel86.img-rhel862.img].log, case_logdir: /root/testconfig/testlogs/2022-04-15/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel86.img-rhel862.img]_15_53_41
2022-04-15 15:53:41 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel86.img-rhel862.img] case ============================
2022-04-15 15:53:41 - test_network_sriov_stress.py - [line:32] - root - INFO - SSH SUT and setup configurations in the setup function.
2022-04-15 15:53:41 - test_network_sriov_stress.py - [line:50] - root - INFO - ssh to the sut1
2022-04-15 15:53:41 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-15 15:53:41 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-15 15:53:41 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.80
2022-04-15 15:53:41 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-15 15:53:41 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-15 15:53:41 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.88
2022-04-15 15:53:41 - test_network_sriov_stress.py - [line:64] - root - INFO - clean up iperf and taskset process...
2022-04-15 15:53:44 - sshInstance.py - [line:50] - root - INFO - pkill -9 iperf
2022-04-15 15:53:44 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:53:52 - sshInstance.py - [line:50] - root - INFO - pkill -9 taskset
2022-04-15 15:53:52 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:54:00 - sshInstance.py - [line:50] - root - INFO - pkill -9 iperf
2022-04-15 15:54:00 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:54:08 - sshInstance.py - [line:50] - root - INFO - pkill -9 taskset
2022-04-15 15:54:08 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:54:13 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 3] Opened sftp connection (server version 3)
2022-04-15 15:54:13 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-15 15:54:16 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-15 15:54:17 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:54:22 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-15 15:54:22 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-15 15:54:25 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-15 15:54:35 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:54:40 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-15 15:54:40 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-15 15:54:43 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-15 15:55:19 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:55:24 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-15 15:55:27 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 up
2022-04-15 15:55:27 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:55:35 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens43f0
2022-04-15 15:55:35 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 4.18.0-369.el8.x86_64
firmware-version: 3.10 0x8000ad67 1.3106.0
expansion-rom-version: 
bus-info: 0000:98:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-15 15:55:40 - test_network_sriov_stress.py - [line:72] - root - INFO - trying to get ipv6 addr for pf1
2022-04-15 15:55:43 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0
2022-04-15 15:55:43 - sshInstance.py - [line:54] - root - INFO - ens43f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.10  netmask 255.255.255.255  broadcast 0.0.0.0
        inet6 fe80::6498:6dc9:5596:c914  prefixlen 64  scopeid 0x20<link>
        ether b4:96:91:a0:20:d8  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 26  bytes 3351 (3.2 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-15 15:55:51 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0
2022-04-15 15:55:51 - sshInstance.py - [line:54] - root - INFO - ens785f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.11  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::b696:91ff:fea0:2170  prefixlen 64  scopeid 0x20<link>
        ether b4:96:91:a0:21:70  txqueuelen 1000  (Ethernet)
        RX packets 11902767407  bytes 18020190901737 (16.3 TiB)
        RX errors 0  dropped 888  overruns 0  frame 0
        TX packets 484834440  bytes 36210255870 (33.7 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


------------------------------ Captured log setup ------------------------------
INFO     root:NicTestFixture.py:20 test -------
----------------------------- Captured stdout call -----------------------------
2022-04-15 15:55:56 - test_network_sriov_stress.py - [line:569] - root - INFO - 
Stress Disable and Enable for 1 time when VF is Assigned VM guest.

2022-04-15 15:55:59 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-22.03/usertools/dpdk-devbind.py -s | grep ens43f0 | awk '{print $1;}'
2022-04-15 15:56:00 - sshInstance.py - [line:54] - root - INFO - 0000:98:00.0

2022-04-15 15:56:05 - test_network_sriov_stress.py - [line:123] - root - INFO - pf_id was found: 0000:98:00.0

2022-04-15 15:56:05 - test_network_sriov_stress.py - [line:577] - root - INFO - format_pf_id is 0000\:98\:00.0

2022-04-15 15:56:05 - test_network_sriov_stress.py - [line:580] - root - INFO - pf_id_key: 98
2022-04-15 15:56:05 - test_network_sriov_stress.py - [line:584] - root - INFO - start to assign ip for ens43f0 of SUT1: 192.168.10.10
2022-04-15 15:56:11 - sshInstance.py - [line:99] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket
Register this system with Red Hat Insights: insights-client --register
Create an account or view all your systems at https://red.ht/insights-dashboard
Last login: Fri Apr 15 16:07:03 2022 from 10.112.97.60
[root@npx-auto-SPR-quanta ~]# ifconfig ens43f0 192.168.10.10
[root@npx-auto-SPR-quanta ~]# 
2022-04-15 15:56:14 - test_network_sriov_stress.py - [line:326] - root - INFO - Successfully assigned ip to ethernet: ens43f0
2022-04-15 15:56:14 - test_network_sriov_stress.py - [line:586] - root - INFO - start to assign ip for ens785f0 of SUT2: 192.168.10.11
2022-04-15 15:56:20 - sshInstance.py - [line:99] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Fri Apr 15 16:06:38 2022 from 10.112.97.60
xhost:  unable to open display ""
[root@CYP10RT ~]# ifconfig ens785f0 192.168.10.11
[root@CYP10RT ~]# 
2022-04-15 15:56:23 - test_network_sriov_stress.py - [line:326] - root - INFO - Successfully assigned ip to ethernet: ens785f0
2022-04-15 15:56:23 - test_network_sriov_stress.py - [line:129] - root - INFO - virtulize sut1 pf to 4 vfs
2022-04-15 15:56:26 - sshInstance.py - [line:50] - root - INFO - echo 4 > /sys/class/net/ens43f0/device/sriov_numvfs
2022-04-15 15:56:30 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:56:35 - test_network_sriov_stress.py - [line:138] - root - INFO - Successfully Virtualized PF to VFs.
2022-04-15 15:56:38 - sshInstance.py - [line:50] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-04-15 15:56:38 - sshInstance.py - [line:54] - root - INFO - 98:01.0
98:01.1
98:01.2
98:01.3

2022-04-15 15:56:43 - test_network_sriov_stress.py - [line:152] - root - INFO - Succssfully Gotten vf_id list:

2022-04-15 15:56:46 - sshInstance.py - [line:50] - root - INFO - modprobe vfio-pci

2022-04-15 15:56:46 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 15:56:51 - test_network_sriov_stress.py - [line:172] - root - INFO - start to bind drive in sut1
2022-04-15 15:56:54 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-22.03/usertools/dpdk-devbind.py -b vfio-pci 98:01.0
2022-04-15 15:56:54 - sshInstance.py - [line:54] - root - INFO - Warning: routing table indicates that interface 0000:98:01.0 is active. Not modifying

2022-04-15 15:56:59 - test_network_sriov_stress.py - [line:186] - root - INFO - Successfuly bound VF to vfio-pci
2022-04-15 15:57:02 - sshInstance.py - [line:50] - root - INFO - /opt/APP/utility/DPDK_install/dpdk-22.03/usertools/dpdk-devbind.py -b vfio-pci 98:01.1
2022-04-15 15:57:03 - sshInstance.py - [line:54] - root - INFO - Warning: routing table indicates that interface 0000:98:01.1 is active. Not modifying

2022-04-15 15:57:08 - test_network_sriov_stress.py - [line:186] - root - INFO - Successfuly bound VF to vfio-pci
2022-04-15 15:57:08 - test_network_sriov_stress.py - [line:604] - root - INFO - lspci -Dnn | grep 98:01.1
2022-04-15 15:57:11 - sshInstance.py - [line:50] - root - INFO - lspci -Dnn | grep 98:01.1
2022-04-15 15:57:11 - sshInstance.py - [line:54] - root - INFO - 0000:98:01.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

2022-04-15 15:57:16 - test_network_sriov_stress.py - [line:616] - root - INFO - Successfully get vf device id: 8086:1889
2022-04-15 15:57:16 - test_network_sriov_stress.py - [line:209] - root - INFO - get default network info: virsh net-info default | grep "Active" | awk '{print $2;}'
2022-04-15 15:57:19 - sshInstance.py - [line:50] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-04-15 15:57:19 - sshInstance.py - [line:54] - root - INFO - yes

2022-04-15 15:57:24 - test_network_sriov_stress.py - [line:212] - root - INFO - Default network is active? yes

2022-04-15 15:57:24 - test_network_sriov_stress.py - [line:629] - root - INFO - start to create VMs....
2022-04-15 15:57:24 - test_network_sriov_stress.py - [line:242] - root - INFO - try to create VM with the cmd1: virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_98_01_0 --boot hd --disk path=/home/image/rhel86.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.0.0,port=5900 --noautoconsole

2022-04-15 15:57:28 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host ---vcpus=5 --network network=default --hostdev pci_0000_98_01_0 --boot hd --disk ppath=/home/image/rhel86.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,,listen=0.0.0.0,port=5900 --noautoconsole
2022-04-15 15:57:30 - sshInstance.py - [line:75] - root - INFO - 
Starting install...
2022-04-15 15:57:34 - sshInstance.py - [line:75] - root - INFO - Domain creation completed.
[root@npx-auto-SPR-quanta ~]# 
2022-04-15 15:57:37 - test_network_sriov_stress.py - [line:253] - root - INFO - Successfully created a VM: VMtest1
2022-04-15 15:57:37 - test_network_sriov_stress.py - [line:242] - root - INFO - try to create VM with the cmd1: virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_98_01_1 --boot hd --disk path=/home/image/rhel862.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.0.0,port=5901 --noautoconsole

2022-04-15 15:57:41 - sshInstance.py - [line:75] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host ---vcpus=5 --network network=default --hostdev pci_0000_98_01_1 --boot hd --disk ppath=/home/image/rhel862.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vncc,listen=0.0.0.0,port=5901 --noautoconsole
2022-04-15 15:57:43 - sshInstance.py - [line:75] - root - INFO - 
Starting install...
2022-04-15 15:57:48 - sshInstance.py - [line:75] - root - INFO - Domain creation completed.
[root@npx-auto-SPR-quanta ~]# 
2022-04-15 15:57:51 - test_network_sriov_stress.py - [line:253] - root - INFO - Successfully created a VM: VMtest2
2022-04-15 15:57:54 - sshInstance.py - [line:50] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-04-15 15:57:55 - sshInstance.py - [line:54] - root - INFO - 192.168.122.138/24

2022-04-15 15:58:00 - test_network_sriov_stress.py - [line:269] - root - INFO - Successfully got VMtest1 IP: 192.168.122.138/24

2022-04-15 15:58:03 - sshInstance.py - [line:50] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-04-15 15:58:03 - sshInstance.py - [line:54] - root - INFO - 192.168.122.76/24

2022-04-15 15:58:08 - test_network_sriov_stress.py - [line:269] - root - INFO - Successfully got VMtest2 IP: 192.168.122.76/24

2022-04-15 15:58:08 - test_network_sriov_stress.py - [line:639] - root - INFO - ip of VM1: 192.168.122.138
2022-04-15 15:58:08 - test_network_sriov_stress.py - [line:640] - root - INFO - ip of VM1: 192.168.122.76
2022-04-15 15:58:08 - test_network_sriov_stress.py - [line:645] - root - INFO - Start ssh VM1 and conf VM1
2022-04-15 15:58:12 - sshInstance.py - [line:75] - root - INFO - ping -c 5 192.168.122.138
PING 192.168.122.138 (192.168.122.138) 56(84) bytes of data.
64 bytes from 192.168.122.138: icmp_seq=1 ttl=64 time=0.135 ms
2022-04-15 15:58:13 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.138: icmp_seq=2 ttl=64 time=0.069 ms
2022-04-15 15:58:14 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.138: icmp_seq=3 ttl=64 time=0.056 ms
2022-04-15 15:58:15 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.138: icmp_seq=4 ttl=64 time=0.065 ms
2022-04-15 15:58:16 - sshInstance.py - [line:75] - root - INFO - 64 bytes from 192.168.122.138: icmp_seq=5 ttl=64 time=0.079 ms

--- 192.168.122.138 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4118ms
rtt min/avg/max/mdev = 0.056/0.080/0.135/0.030 ms
[root@npx-auto-SPR-quanta ~]# 
2022-04-15 15:58:25 - sshInstance.py - [line:99] - root - INFO - ssh root@192.168.122.138
The authenticity of host '192.168.122.138 (192.168.122.138)' can't be established.
ECDSA key fingerprint is SHA256:J52wv4LPVprqJsqIIJZZQXHlsApVTL3W7ooy490//CA.
Are you sure you want to continue connecting (yes/no/[fingerprint])? 
2022-04-15 15:58:34 - sshInstance.py - [line:99] - root - INFO - yes
Warning: Permanently added '192.168.122.138' (ECDSA) to the list of known hosts.
root@192.168.122.138's password: 
2022-04-15 15:58:43 - sshInstance.py - [line:99] - root - INFO - 
Activate the web console with: systemctl enable --now cockpit.socket
Register this system with Red Hat Insights: insights-client --register
Create an account or view all your systems at https://red.ht/insights-dashboard
Last login: Thu Apr 14 22:19:50 2022 from 192.168.122.1
[root@philliph-IVM ~]# 
2022-04-15 15:58:46 - test_network_sriov_stress.py - [line:352] - root - INFO - Successfully ssh VM.
2022-04-15 15:58:52 - sshInstance.py - [line:99] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $$1;}'
0000:05:00.0
[root@philliph-IVM ~]# 
2022-04-15 15:58:58 - test_network_sriov_stress.py - [line:369] - root - INFO - successfully got vf pci id of vm1: 0000:05:00.0
2022-04-15 15:59:04 - sshInstance.py - [line:99] - root - INFO - ls /sys/bus/pci/devices/0000\:05\:00.0/net/ --color=never [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            r
enp5s0
[root@philliph-IVM ~]# 
2022-04-15 15:59:09 - test_network_sriov_stress.py - [line:383] - root - INFO - Successfully got vf device name of vm1: enp5s0
2022-04-15 15:59:15 - sshInstance.py - [line:99] - root - INFO - ifconfig enp5s0 192.168.10.12
[root@philliph-IVM ~]# 
2022-04-15 15:59:18 - test_network_sriov_stress.py - [line:326] - root - INFO - Successfully assigned ip to ethernet: enp5s0
2022-04-15 15:59:24 - sshInstance.py - [line:99] - root - INFO - ifconfig enp5s0 down
[root@philliph-IVM ~]# 
2022-04-15 15:59:33 - sshInstance.py - [line:99] - root - INFO - ifconfig enp5s0 hw ether 1234567890ac
[root@philliph-IVM ~]# 
2022-04-15 15:59:42 - sshInstance.py - [line:99] - root - INFO - ifconfig enp5s0 up
[root@philliph-IVM ~]# 
2022-04-15 15:59:51 - sshInstance.py - [line:99] - root - INFO - nmcli device connect enp5s0
[2KDevice 'enp5s0' successfully activated with '1f069b77-7a05-45a6-bec1-3e45ca49d66e'.
[root@philliph-IVM ~]# 
2022-04-15 16:00:00 - sshInstance.py - [line:99] - root - INFO - ifconfig enp5s0
enp5s0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 27  bytes 2826 (2.7 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
[root@philliph-IVM ~]# 
2022-04-15 16:00:09 - sshInstance.py - [line:99] - root - INFO - exit
logout
Connection to 192.168.122.138 closed.
[root@npx-auto-SPR-quanta ~]# 
------------------------------ Captured log call -------------------------------
INFO     root:test_network_sriov_stress.py:569 
Stress Disable and Enable for 1 time when VF is Assigned VM guest.

INFO     root:sshInstance.py:50 /opt/APP/utility/DPDK_install/dpdk-22.03/usertools/dpdk-devbind.py -s | grep ens43f0 | awk '{print $1;}'
INFO     root:sshInstance.py:54 0000:98:00.0

INFO     root:test_network_sriov_stress.py:123 pf_id was found: 0000:98:00.0

INFO     root:test_network_sriov_stress.py:577 format_pf_id is 0000\:98\:00.0

INFO     root:test_network_sriov_stress.py:580 pf_id_key: 98
INFO     root:test_network_sriov_stress.py:584 start to assign ip for ens43f0 of SUT1: 192.168.10.10
INFO     root:sshInstance.py:99 Activate the web console with: systemctl enable --now cockpit.socket
Register this system with Red Hat Insights: insights-client --register
Create an account or view all your systems at https://red.ht/insights-dashboard
Last login: Fri Apr 15 16:07:03 2022 from 10.112.97.60
[root@npx-auto-SPR-quanta ~]# ifconfig ens43f0 192.168.10.10
[root@npx-auto-SPR-quanta ~]# 
INFO     root:test_network_sriov_stress.py:326 Successfully assigned ip to ethernet: ens43f0
INFO     root:test_network_sriov_stress.py:586 start to assign ip for ens785f0 of SUT2: 192.168.10.11
INFO     root:sshInstance.py:99 Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Fri Apr 15 16:06:38 2022 from 10.112.97.60
xhost:  unable to open display ""
[root@CYP10RT ~]# ifconfig ens785f0 192.168.10.11
[root@CYP10RT ~]# 
INFO     root:test_network_sriov_stress.py:326 Successfully assigned ip to ethernet: ens785f0
INFO     root:test_network_sriov_stress.py:129 virtulize sut1 pf to 4 vfs
INFO     root:sshInstance.py:50 echo 4 > /sys/class/net/ens43f0/device/sriov_numvfs
INFO     root:sshInstance.py:54 
INFO     root:test_network_sriov_stress.py:138 Successfully Virtualized PF to VFs.
INFO     root:sshInstance.py:50  lspci | grep "Virtual" | awk '{print $1;}'list
INFO     root:sshInstance.py:54 98:01.0
98:01.1
98:01.2
98:01.3

INFO     root:test_network_sriov_stress.py:152 Succssfully Gotten vf_id list:

INFO     root:sshInstance.py:50 modprobe vfio-pci

INFO     root:sshInstance.py:54 
INFO     root:test_network_sriov_stress.py:172 start to bind drive in sut1
INFO     root:sshInstance.py:50 /opt/APP/utility/DPDK_install/dpdk-22.03/usertools/dpdk-devbind.py -b vfio-pci 98:01.0
INFO     root:sshInstance.py:54 Warning: routing table indicates that interface 0000:98:01.0 is active. Not modifying

INFO     root:test_network_sriov_stress.py:186 Successfuly bound VF to vfio-pci
INFO     root:sshInstance.py:50 /opt/APP/utility/DPDK_install/dpdk-22.03/usertools/dpdk-devbind.py -b vfio-pci 98:01.1
INFO     root:sshInstance.py:54 Warning: routing table indicates that interface 0000:98:01.1 is active. Not modifying

INFO     root:test_network_sriov_stress.py:186 Successfuly bound VF to vfio-pci
INFO     root:test_network_sriov_stress.py:604 lspci -Dnn | grep 98:01.1
INFO     root:sshInstance.py:50 lspci -Dnn | grep 98:01.1
INFO     root:sshInstance.py:54 0000:98:01.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889] (rev 02)

INFO     root:test_network_sriov_stress.py:616 Successfully get vf device id: 8086:1889
INFO     root:test_network_sriov_stress.py:209 get default network info: virsh net-info default | grep "Active" | awk '{print $2;}'
INFO     root:sshInstance.py:50 virsh net-info default | grep "Active" | awk '{print $2;}'
INFO     root:sshInstance.py:54 yes

INFO     root:test_network_sriov_stress.py:212 Default network is active? yes

INFO     root:test_network_sriov_stress.py:629 start to create VMs....
INFO     root:test_network_sriov_stress.py:242 try to create VM with the cmd1: virt-install --name VMtest1 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_98_01_0 --boot hd --disk path=/home/image/rhel86.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.0.0,port=5900 --noautoconsole

INFO     root:sshInstance.py:75 virt-install --name VMtest1 --ram 4096 --cpu host ---vcpus=5 --network network=default --hostdev pci_0000_98_01_0 --boot hd --disk ppath=/home/image/rhel86.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,,listen=0.0.0.0,port=5900 --noautoconsole
INFO     root:sshInstance.py:75 
Starting install...
INFO     root:sshInstance.py:75 Domain creation completed.
[root@npx-auto-SPR-quanta ~]# 
INFO     root:test_network_sriov_stress.py:253 Successfully created a VM: VMtest1
INFO     root:test_network_sriov_stress.py:242 try to create VM with the cmd1: virt-install --name VMtest2 --ram 4096 --cpu host --vcpus=5 --network network=default --hostdev pci_0000_98_01_1 --boot hd --disk path=/home/image/rhel862.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vnc,listen=0.0.0.0,port=5901 --noautoconsole

INFO     root:sshInstance.py:75 virt-install --name VMtest2 --ram 4096 --cpu host ---vcpus=5 --network network=default --hostdev pci_0000_98_01_1 --boot hd --disk ppath=/home/image/rhel862.img,size=50,format=qcow2 --osinfo rhel8.2 --graphics vncc,listen=0.0.0.0,port=5901 --noautoconsole
INFO     root:sshInstance.py:75 
Starting install...
INFO     root:sshInstance.py:75 Domain creation completed.
[root@npx-auto-SPR-quanta ~]# 
INFO     root:test_network_sriov_stress.py:253 Successfully created a VM: VMtest2
INFO     root:sshInstance.py:50 virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
INFO     root:sshInstance.py:54 192.168.122.138/24

INFO     root:test_network_sriov_stress.py:269 Successfully got VMtest1 IP: 192.168.122.138/24

INFO     root:sshInstance.py:50 virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
INFO     root:sshInstance.py:54 192.168.122.76/24

INFO     root:test_network_sriov_stress.py:269 Successfully got VMtest2 IP: 192.168.122.76/24

INFO     root:test_network_sriov_stress.py:639 ip of VM1: 192.168.122.138
INFO     root:test_network_sriov_stress.py:640 ip of VM1: 192.168.122.76
INFO     root:test_network_sriov_stress.py:645 Start ssh VM1 and conf VM1
INFO     root:sshInstance.py:75 ping -c 5 192.168.122.138
PING 192.168.122.138 (192.168.122.138) 56(84) bytes of data.
64 bytes from 192.168.122.138: icmp_seq=1 ttl=64 time=0.135 ms
INFO     root:sshInstance.py:75 64 bytes from 192.168.122.138: icmp_seq=2 ttl=64 time=0.069 ms
INFO     root:sshInstance.py:75 64 bytes from 192.168.122.138: icmp_seq=3 ttl=64 time=0.056 ms
INFO     root:sshInstance.py:75 64 bytes from 192.168.122.138: icmp_seq=4 ttl=64 time=0.065 ms
INFO     root:sshInstance.py:75 64 bytes from 192.168.122.138: icmp_seq=5 ttl=64 time=0.079 ms

--- 192.168.122.138 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4118ms
rtt min/avg/max/mdev = 0.056/0.080/0.135/0.030 ms
[root@npx-auto-SPR-quanta ~]# 
INFO     root:sshInstance.py:99 ssh root@192.168.122.138
The authenticity of host '192.168.122.138 (192.168.122.138)' can't be established.
ECDSA key fingerprint is SHA256:J52wv4LPVprqJsqIIJZZQXHlsApVTL3W7ooy490//CA.
Are you sure you want to continue connecting (yes/no/[fingerprint])? 
INFO     root:sshInstance.py:99 yes
Warning: Permanently added '192.168.122.138' (ECDSA) to the list of known hosts.
root@192.168.122.138's password: 
INFO     root:sshInstance.py:99 
Activate the web console with: systemctl enable --now cockpit.socket
Register this system with Red Hat Insights: insights-client --register
Create an account or view all your systems at https://red.ht/insights-dashboard
Last login: Thu Apr 14 22:19:50 2022 from 192.168.122.1
[root@philliph-IVM ~]# 
INFO     root:test_network_sriov_stress.py:352 Successfully ssh VM.
INFO     root:sshInstance.py:99 lspci -Dnn | grep "8086:1889" --color=never| awk '{print $$1;}'
0000:05:00.0
[root@philliph-IVM ~]# 
INFO     root:test_network_sriov_stress.py:369 successfully got vf pci id of vm1: 0000:05:00.0
INFO     root:sshInstance.py:99 ls /sys/bus/pci/devices/0000\:05\:00.0/net/ --color=never [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            r
enp5s0
[root@philliph-IVM ~]# 
INFO     root:test_network_sriov_stress.py:383 Successfully got vf device name of vm1: enp5s0
INFO     root:sshInstance.py:99 ifconfig enp5s0 192.168.10.12
[root@philliph-IVM ~]# 
INFO     root:test_network_sriov_stress.py:326 Successfully assigned ip to ethernet: enp5s0
INFO     root:sshInstance.py:99 ifconfig enp5s0 down
[root@philliph-IVM ~]# 
INFO     root:sshInstance.py:99 ifconfig enp5s0 hw ether 1234567890ac
[root@philliph-IVM ~]# 
INFO     root:sshInstance.py:99 ifconfig enp5s0 up
[root@philliph-IVM ~]# 
INFO     root:sshInstance.py:99 nmcli device connect enp5s0
[2KDevice 'enp5s0' successfully activated with '1f069b77-7a05-45a6-bec1-3e45ca49d66e'.
[root@philliph-IVM ~]# 
INFO     root:sshInstance.py:99 ifconfig enp5s0
enp5s0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 27  bytes 2826 (2.7 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
[root@philliph-IVM ~]# 
INFO     root:sshInstance.py:99 exit
logout
Connection to 192.168.122.138 closed.
[root@npx-auto-SPR-quanta ~]#
_____________ Test_Nics.test_gold_PI_Networking_SRIOV_CheckDevice ______________

self = <Tests.testNics.test_nics.Test_Nics object at 0x7f9a8e35e438>

    @pytest.mark.usefixtures("fbefore")
    @pytest.mark.nicAll
    @pytest.mark.nicGold
    @pytest.mark.nicSilver
    @pytest.mark.nicSimple
    @pytest.mark.test
    def test_gold_PI_Networking_SRIOV_CheckDevice(self):
        cmd = "echo 4 > /sys/class/net/{}/device/sriov_numvfs".format(self.sut_conf["pf1"])
        stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client, cmd)
        time.sleep(5)
        if stdout or stderr:
            assert 200 == 400
        sut2_ip = "192.168.10.11"
        self.cn_test_sut.ssh_exec_cmd(self.client2, "ifconfig {} {}\n".format(self.pkg_generator["pf1"], sut2_ip))
        port_up_cmd = "ifconfig {} up"
        #manual_ip_cmd = "ifconfig {}{} 192.168.{}.{}"
        manual_ip_cmd = "ifconfig {}{} 192.168.{}.{}"
        ping_ipv4_cmd = "ping -c 5 192.168.{}.{}"
        ping_ipv6_cmd = "ping6 -c 5 -I {}{} {}"
        for i in range(4):
            vf = "v"+str(i)
            ip_part = str(int(sut2_ip.split(".")[3])+i+1)
            time.sleep(5)
            stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client, "ifconfig {}{} down".format(self.sut_conf["pf1"], vf))
            stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client, "ifconfig {}{} up".format(self.sut_conf["pf1"], vf))
            stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client, manual_ip_cmd.format(self.sut_conf["pf1"], vf,sut2_ip.split(".")[2] ,ip_part))
            time.sleep(3)
            if stdout or stderr:
                assert 200 == 400
            stdout, stderr = self.cn_test_sut.ssh_exec_cmd(self.client2, ping_ipv4_cmd.format(sut2_ip.split(".")[2], ip_part))
            time.sleep(3)
            if stdout.find("received, 0% packet loss") == -1:
>               assert 200 == 400
E               assert 200 == 400

npxtester/Tests/testNics/test_nics.py:186: AssertionError
---------------------------- Captured stdout setup -----------------------------
2022-04-15 16:01:41 - NicTestFixture.py - [line:20] - root - INFO - test -------
2022-04-15 16:01:41 - NicTestFixture.py - [line:27] - root - INFO - case_logpath: /root/testconfig/testlogs/2022-04-15/test_gold_PI_Networking_SRIOV_CheckDevice_16_01_41/test_gold_PI_Networking_SRIOV_CheckDevice.log, case_logdir: /root/testconfig/testlogs/2022-04-15/test_gold_PI_Networking_SRIOV_CheckDevice_16_01_41
2022-04-15 16:01:41 - NicTestFixture.py - [line:31] - root - INFO - ============================ Starting to run test_gold_PI_Networking_SRIOV_CheckDevice case ============================
2022-04-15 16:01:41 - test_nics.py - [line:21] - root - INFO - this is Test_Nics setup.
2022-04-15 16:01:41 - test_nics.py - [line:28] - root - INFO - ssh to the sut1
2022-04-15 16:01:41 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-15 16:01:41 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-15 16:01:41 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.80
2022-04-15 16:01:41 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-04-15 16:01:41 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-04-15 16:01:41 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.182.88
2022-04-15 16:01:41 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-04-15 16:01:44 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 up
2022-04-15 16:01:44 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:01:52 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens43f0 | grep ^version:
2022-04-15 16:01:52 - sshInstance.py - [line:54] - root - INFO - version: 4.18.0-369.el8.x86_64

2022-04-15 16:01:57 - test_nics.py - [line:58] - root - INFO - Passed: driver version is what expected.
2022-04-15 16:01:57 - sshInstance.py - [line:190] - root - INFO - start to rmmod irdma....
2022-04-15 16:02:00 - sshInstance.py - [line:50] - root - INFO - rmmod irdma
2022-04-15 16:02:00 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:02:05 - sshInstance.py - [line:193] - root - INFO - finished rmmod irdma....
2022-04-15 16:02:05 - sshInstance.py - [line:194] - root - INFO - start to rmmod ice....
2022-04-15 16:02:08 - sshInstance.py - [line:50] - root - INFO - rmmod ice
2022-04-15 16:02:19 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:02:24 - sshInstance.py - [line:197] - root - INFO - finished rmmod ice....
2022-04-15 16:02:24 - sshInstance.py - [line:199] - root - INFO - start to modprobe ice....
2022-04-15 16:02:27 - sshInstance.py - [line:50] - root - INFO - modprobe ice
2022-04-15 16:03:02 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:03:07 - sshInstance.py - [line:201] - root - INFO - finished modprobe ice....
2022-04-15 16:03:10 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0 up
2022-04-15 16:03:10 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:03:18 - sshInstance.py - [line:50] - root - INFO - ethtool -i ens43f0
2022-04-15 16:03:18 - sshInstance.py - [line:54] - root - INFO - driver: ice
version: 4.18.0-369.el8.x86_64
firmware-version: 3.10 0x8000ad67 1.3106.0
expansion-rom-version: 
bus-info: 0000:98:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-04-15 16:03:26 - sshInstance.py - [line:50] - root - INFO - modprobe iavf
2022-04-15 16:03:27 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:03:32 - test_nics.py - [line:65] - root - INFO - --Finish modprobe iavf --
2022-04-15 16:03:35 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0
2022-04-15 16:03:35 - sshInstance.py - [line:54] - root - INFO - ens43f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.10  netmask 255.255.255.255  broadcast 0.0.0.0
        inet6 fe80::6498:6dc9:5596:c914  prefixlen 64  scopeid 0x20<link>
        ether b4:96:91:a0:20:d8  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 22  bytes 3024 (2.9 KiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-04-15 16:03:43 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0
2022-04-15 16:03:43 - sshInstance.py - [line:54] - root - INFO - ens785f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::b696:91ff:fea0:2170  prefixlen 64  scopeid 0x20<link>
        ether b4:96:91:a0:21:70  txqueuelen 1000  (Ethernet)
        RX packets 11902767571  bytes 18020190921635 (16.3 TiB)
        RX errors 0  dropped 908  overruns 0  frame 0
        TX packets 484834442  bytes 36210255990 (33.7 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


------------------------------ Captured log setup ------------------------------
INFO     root:NicTestFixture.py:20 test -------
----------------------------- Captured stdout call -----------------------------
2022-04-15 16:03:51 - sshInstance.py - [line:50] - root - INFO - echo 4 > /sys/class/net/ens43f0/device/sriov_numvfs
2022-04-15 16:03:55 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:04:08 - sshInstance.py - [line:50] - root - INFO - ifconfig ens785f0 192.168.10.11

2022-04-15 16:04:08 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:04:21 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0v0 down
2022-04-15 16:04:21 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:04:29 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0v0 up
2022-04-15 16:04:29 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:04:37 - sshInstance.py - [line:50] - root - INFO - ifconfig ens43f0v0 192.168.10.12
2022-04-15 16:04:37 - sshInstance.py - [line:54] - root - INFO - 
2022-04-15 16:04:48 - sshInstance.py - [line:50] - root - INFO - ping -c 5 192.168.10.12
2022-04-15 16:05:03 - sshInstance.py - [line:54] - root - INFO - PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 0 received, 100% packet loss, time 4129ms


------------------------------ Captured log call -------------------------------
INFO     root:sshInstance.py:50 echo 4 > /sys/class/net/ens43f0/device/sriov_numvfs
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens785f0 192.168.10.11

INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens43f0v0 down
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens43f0v0 up
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ifconfig ens43f0v0 192.168.10.12
INFO     root:sshInstance.py:54 
INFO     root:sshInstance.py:50 ping -c 5 192.168.10.12
INFO     root:sshInstance.py:54 PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 0 received, 100% packet loss, time 4129ms
=============================== warnings summary ===============================
npxtester/Tests/testNics/test_iperf_client_4hour.py:88
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_iperf_client_4hour.py:88: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_tcp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_tcp

npxtester/Tests/testNics/test_iperf_client_4hour.py:137
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_iperf_client_4hour.py:137: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_udp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_udp

npxtester/Tests/testNics/test_mtu.py:69
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_mtu.py:69: PytestUnknownMarkWarning: Unknown pytest.mark.two_sut_CVL - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.two_sut_CVL

npxtester/Tests/testNics/test_mtu.py:70
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_mtu.py:70: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_mtu.py:109
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_mtu.py:109: PytestUnknownMarkWarning: Unknown pytest.mark.two_sut_CVL - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.two_sut_CVL

npxtester/Tests/testNics/test_mtu.py:110
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_mtu.py:110: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_network_sriov_stress.py:565
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_network_sriov_stress.py:565: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov5times - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov5times

npxtester/Tests/testNics/test_network_sriov_stress.py:691
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_network_sriov_stress.py:691: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov2hours - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov2hours

npxtester/Tests/testNics/test_network_sriov_stress.py:931
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_network_sriov_stress.py:931: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_network_sriov_stress.py:1067
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_network_sriov_stress.py:1067: PytestUnknownMarkWarning: Unknown pytest.mark.nicIperfTcpSUTGuestStress - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicIperfTcpSUTGuestStress

npxtester/Tests/testNics/test_network_sriov_stress.py:1068
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_network_sriov_stress.py:1068: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_nics.py:102
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_nics.py:102: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:128
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_nics.py:128: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:158
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_nics.py:158: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:159
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_nics.py:159: PytestUnknownMarkWarning: Unknown pytest.mark.test - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.test

npxtester/Tests/testNics/test_nics.py:246
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_nics.py:246: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:272
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_nics.py:272: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:315
  /root/.jenkins/workspace/spr-test-nicGold/npxtester/Tests/testNics/test_nics.py:315: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

-- Docs: https://docs.pytest.org/en/stable/warnings.html
- generated html file: file:///root/testconfig/testlogs/HtmlReport/nicGold_report.html -
=========================== short test summary info ============================
FAILED npxtester/Tests/testNics/test_network_sriov_stress.py::Test_Network_SRIOV_Stress::test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[rhel86.img-rhel862.img]
FAILED npxtester/Tests/testNics/test_nics.py::Test_Nics::test_gold_PI_Networking_SRIOV_CheckDevice
===== 2 failed, 2 passed, 15 deselected, 18 warnings in 1097.96s (0:18:17) =====
[8mha:////4DDaF6xD1n97fCJC17HEPtC4MvFSxjBMoUPdpyGs71EZAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIioEIoFa3lhheYxBgn1l2wHZyKF/E1/oBFJCq2WO1sM683LIOHmr3BTlNvKTQWBzeavDCx76+OE3Z8wYYpsNModZLc6tOMkqOGOUUJCwErTY3jYMlEWItOPVTlFJnqHH3+jgJK297hCYXI5qh8/MLkYXMb28D08/wVwDREKHeHXPvt8AGMzKKgvgAAAA==[0m[Pipeline] echo
1
[8mha:////4JdwCG2vyNU1vR63Cq0h1I9fweuZdBzoUYFbNZMa07ZpAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIihQohqrRWGl5gYmOcWHfGdkgqXsTX+AMWkajYYrWzzbzesI4BThwM9poGS7Gz6N1o8sKJw3B1PGHPF+yYIjuNrZ5aVrpZsOWkYUlRwkrARlPnOFoyCbailw9ZOUmmOqeQv6OA0qo7PKEQ2ZxkSF+YA+xuo4pMP89fAcw+QVkfcu1r/wFPn4+LvgAAAA==[0m[Pipeline] sh
+ sed -i $'s/\r//g' /root/.jenkins/workspace/spr-test-nicGold/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4J0lAWMm8HuBR5s3oE3lokuIt9KB2+AHQUkjEiJ+aQGpAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPMKRSIUQVVorDS8w8WGcWHfGdkgqXsTX+AMWkajYYrWzzbzesI4BThyM6JEGS7GzwrvR5CUmDsPV8SR6voiOKbJD0eLUssZmwZYTwpKihJWEDVLnOFoyCbayVw9VOUWmOqeQv6OE0uo7PKGQ2ZxUSF+YA+xuo45MP89fAcw+Qbk/5Kpr/wEOrpSSvgAAAA==[0m[Pipeline] sh
+ chmod a+x /root/.jenkins/workspace/spr-test-nicGold/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4Mt0DSsxkJzZfVcF7hGhEOllvDy/ttdaA/Rf0YiZvcahAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiAQ1CqWgtN7zAJMY4se6C7eBUvIiv8QcsIlGxxWpnm3m9YRk81OwNdpp6S6GxOLjR5IWJfX91nLDjCzZMgZ1GqZPkVp9mlBw1zClKWAhYaWocB0smwlp06qEqp8hU5+jzdxRQ2vYOTyhENkfl4xcmD5vb2Aamn+evAKYhQrk95Nrthw/JONXdvgAAAA==[0m[Pipeline] sh
+ sh /root/.jenkins/workspace/spr-test-nicGold/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4BJ3niM699yfMaO4XlYckMNVxHBxF/A/6tXrT01aXGOTAAAAph+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigSgQSkVrueEFJjHGiXUXbAen4kV8jT9gEYmKLVY728zrDcvgoWZvsNPUWwqNxcGNJi9M7Pur44QdX7BhCuw0Sp0kt/o0o+SoYU5RwkLASlPjOFgyEdaiUw9VOUWmOkefv6OA0rZ3eEIhsjkqH78wedjcxjYw/Tx/BTANQ4Ryt8+1PXwARMxJg74AAAA=[0m[Pipeline] }
[8mha:////4KLOGR+83vgrLFidGueNeEu4h+87+yS5qW1q0kigPKEkAAAAph+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigYSQUCpayw0vMIkxTqy7YDs4FS/ia/wBi0hUbLHa2WZeb1gGDzV7g52m3lJoLA5uNHlhYt9fHSfs+IINU2CnUeokudWnGSVHDXOKEhYCVpoax8GSibAWnXqoyiky1Tn6/B0FlLa9wxMKkc1R+fiFycPmNraB6ef5K4BpGCKUu32u7eEDfI7acr4AAAA=[0m[Pipeline] // script
[8mha:////4B/bjeg5xruNbbx3kITLupXWvVAJcKllvbrOrw34K5MTAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiQZECUaW10vACExvjxLoztkNS8SK+xh+wiETFFqudbeb1hnUMcOJgsNc0WIqdRe9GkxdOHIar4wl7vmDHFNlpbPXUstLNgi0nDUuKElYCNpo6x9GSSbAVvXzIykky1TmF/B0FlFbd4QmFyOYkQ/rCHGB3G1Vk+nn+CmD2PkF5qHPt6w9IQDkpvgAAAA==[0m[Pipeline] }
[8mha:////4Pysn2p6Cuaw38A5WKCzX3hZbaQrVsudvseeZZRTcVjBAAAAph+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigYSQUCpayw0vMIkxTqy7YDs4FS/ia/wBi0hUbLHa2WZeb1gGDzV7g52m3lJoLA5uNHlhYt9fHSfs+IINU2CnUeokudWnGSVHDXOKEhYCVpoax8GSibAWnXqoyiky1Tn6/B0FlLa9wxMKkc1R+fiFycPmNraB6ef5K4BpGCKUu0Ou7f4DC8Jor74AAAA=[0m[Pipeline] // stage
[8mha:////4AtNwyaW/AZqBd3OKxZrkWfAC6zB3HxngMFQ9sJmqFs1AAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIpoEGIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsXZLsP/+idkr0AAAA=[0m[Pipeline] }
[8mha:////4I5dJP3UyABwbWbQZBRGWAtgfXC/GfHsQ3SW1rlV0+rIAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMJBgg5R0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYu06Q77eCbPcBlS/sHL0AAAA=[0m[Pipeline] // node
[8mha:////4OCxxHdXTRYePL6pNSOSjI/nVmiKF6Ardumg4UkWHzykAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRiA5R0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYu06Q77eCbPcBszaLVr0AAAA=[0m[Pipeline] End of Pipeline
Finished: FAILURE
